neat-form-core/src/main/java/com/nerdstone/neatformcore/views/builders/CheckBoxViewBuilder.kt

```diff
@@ -14,7 +14,7 @@ class CheckBoxViewBuilder(override val nFormView: NFormView) : ViewBuilder {
     override val acceptedAttributes = Utils.convertEnumToSet(CheckBoxProperties::class.java)
 
     enum class CheckBoxProperties {
-        TEXT
+        TEXT, CHECK_BOX_TEXT_SIZE
```

Comment:
we can just have CHECK_BOX_TEXT_SIZE as TEXT_SIZE to reduce the verbosity and maintain the convention or mapping android xml properties to json attributes